webui/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "webui-react",
-  "version": "0.1.0",
+  "name": "determined-webui",
+  "version": "1.0.0",
```

Comment:
I think we can call it `1.0.0` at this point :)

Reply:
great idea

Reply:
doh, `package.json` will not accept a version such as `0.14.7.dev0`, it has to conform to `[major].[minor].[patch]`